templates/maas/features.html

```diff
@@ -0,0 +1,526 @@
+          aria-label="Link to the install config images page on MAAS documentation">Learn more about images&nbsp;&rsaquo;</a>'
+      },
+      {
+        "title_text": 'Deploy with the press of a button',
+        "image_html": lightbox_image(
+          img="https://assets.ubuntu.com/v1/3b72e050-deploy_with_1.jpg",
+          description="node deployment",
+        ),
+        "description_html": "<ol>
+            <li>Select the machine</li>
+            <li>Choose the OS and architecture</li>
+            <li>Press 'deploy'</li>
+          </ol>",
+        "cta_html": '<a href="/maas/docs/how-to-manage-machines#p-9078-deploy-machines"
+          onclick="dataLayer.push({&quot;event&quot;: &quot;GAEvent&quot;, &quot;eventCategory&quot;: &quot;Feature&quot;, &quot;eventAction&quot;: &quot;Click&quot;, &quot;eventLabel&quot;: &quot;Fast deployment&quot; });"
+          aria-label="Link to the install and config page of node deployment on MAAS documentation">Learn more about
+          deployment&nbsp;&rsaquo;</a>'
+      },
+      {
+        "title_text": 'Get all the information you need',
+        "image_html": lightbox_image(
+          img="https://assets.ubuntu.com/v1/4e398f57-get_all_the_1.jpg",
+          description="commissioning list",
+        ),
+        "description_html": "<ul>
+          <li>
+            Dig into your hardware details that MAAS gathers during commissioning including; processors, memory, storage,
+            networking, and other components.
+          </li>
+          <li>Event tracking allows you to monitor your machine throughout its lifecycle.</li>
+        </ul>",
+        "cta_html": '<a href="/maas/docs/about-commissioning-machines"
+          aria-label="Link to the about commissioning machines page on MAAS documentation">Learn more about commissioning machines&nbsp;&rsaquo;</a>'
+      },
+    ]
+  ) %}
+    {% if slot == "description" %}
+    <p>
+      Use the Web <abbr title="User interface">UI</abbr> or the command line (<abbr title="Command-line interface">CLI</abbr>)
+      to remotely manage your nodes or use the <abbr title="Application programming interface">API</abbr> to automate
+      management.
+    </p>
+    {% endif %}
+  {% endcall %}
+  {#- The empty title text params can be removed once the following PR has been released: https://github.com/canonical/vanilla-framework/pull/5666 -#}
+  {% call(slot) vf_equal_heights(
+    title_text="Keep an eye on your hardware",
+    image_aspect_ratio_small="auto",
+    image_aspect_ratio_medium="auto",
+    image_aspect_ratio_large="auto",
+    items=[
+      {
+        "title_text": '',
+        "image_html": lightbox_image(
+          img="https://assets.ubuntu.com/v1/c95d1bf2-check_your_data_1.jpg",
+          description="node listing",
+        ),
+        "description_html": '<p style="margin-top: -1rem">Check your data centre health at a glance and easily identify faulty components.</p>',
```

Comment:
```suggestion
        "description_html": '<p style="margin-top: -1rem">Check your data center health at a glance and easily identify faulty components.</p>',
```

Use American English, as in the copydoc